Makefile

```diff
@@ -256,6 +256,10 @@ update_pinpoint_supported_countries: ## Updates list of countries supported by P
 lint_country_dialing_codes: update_pinpoint_supported_countries ## Checks that countries supported by Pinpoint for voice and SMS are up to date
 	(! git diff --name-only | grep config/country_dialing_codes.yml) || (echo "Error: Run 'make update_pinpoint_supported_countries' to update country codes"; exit 1)
 
+lint_database_schema_files:
```

Comment:
optional bonus points for adding to `.PHONY` list and add a comment via ` ## ` syntax

Reply:
Thanks for catching that, added